microsoft365defender.json

```diff
@@ -45,11 +45,32 @@
             "required": true,
             "order": 3
         },
+        "timeout": {
+            "description": "HTTP API timeout in seconds",
+            "data_type": "numeric",
+            "order": 4
+        },
         "non_interactive": {
             "description": "Non-Interactive Auth",
             "data_type": "boolean",
             "default": false,
-            "order": 4
+            "order": 5
+        },
+        "max_incidents_for_polling": {
```

Comment:
Maybe `per_poll` instead of `for_polling`?

Reply:
changed